pkgs/by-name/vm/vmctl/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  bash,
+  shellcheck,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "vmctl";
+  version = "1";
+
+  src = fetchFromGitHub {
+    owner = "SamsungDS";
+    repo = "vmctl";
+    rev = "5b6b7084b8cba06b474c0e020df965237f2c832c";
+    sha256 = "sha256-yDgaY2RJXBjWkMSQb4JaJzMGLFzowfOGixSRzzv2iIk=";
+  };
+
+  nativeBuildInputs = [
+    bash
```

Comment:
Could you elaborate on the reason behind putting bash here? It's unusual to list bash as a native input

Reply:
It is a bash script. Actually, bash is more like a runtime dependency and not a build input. So can we just assume it is present instead of explicitly mentioning it?

Reply:
That would mean it's not native (i.e. if you were cross building from x64 to arm64, you'd want to end up with the arm64 bash, not the x64 one), and it also happens to get fixed by default in the `fixupPhase` where there is a script that replaces the shebangs with nix-specific ones. You can probably just drop it :)